docs/_future/app_manifests.md

```diff
@@ -82,9 +82,9 @@ The properties in the Manifest are:
 | `longDescription` | String | ✅ Yes | (Optional) A more detailed description of your application |
 | `icon` | String | ❌ No | A relative path to an image asset to use for the app's icon. Your app's profile picture that will appear in the Slack client |
 | `backgroundColor` | String | ✅ Yes | (Optional) A six digit combination of numbers and letters (the hexadecimal color code) that make up the color of your app background e.g., "#000000" is the color black |
-| `botScopes` | Array<string> | ✅ Yes | A list of [scopes](/scopes), or permissions, the app's Functions require |
-| `functions` | Array | ✅ Yes | (Optional) A list of all Functions your app will use |
-| `workflows` | Array | ✅ Yes | (Optional) A list of all Workflows your app will use |
+| `botScopes` | Array<string> | ✅ Yes | A list of [scopes](/scopes), or permissions, the app's functions require |
+| `functions` | Array | ✅ Yes | (Optional) A list of all functions your app will use |
+| `workflows` | Array | ✅ Yes | (Optional) A list of all workflows your app will use |
```

Comment:
I don't seem to be able to drop a comment on ln91 but "Datastores" seems like it would also be lowercased right?

Reply:
And same thing for Manifest in ln94?